site/src/pages/docs/client-factory-builder.mdx

```diff
@@ -0,0 +1,139 @@
+which is desirable for typical HTTP/2 services. Armeria achieves this by assigning one `EventLoop` per endpoint
+from the <type://ServerConfig#workerGroup()>. However, there's a limit to the requests that
+a single connection(or `EventLoop` which is a thread) can handle, so you might want to increase
+the number of connections(or `EventLoop` threads) for certain or all endpoints.
+
+You can do this via <type://ClientFactoryBuilder#maxNumEventLoopsPerEndpoint(int)>:
+
+The <type://ClientFactoryBuilder> class has a feature to set the number of `EventLoop` threads using
+`maxNumEventLoopsPerEndpoint` and `maxNumEventLoopsFunction`. The former parameter will set the number of `EventLoop`
+threads for each endpoint (This parameter is used for HTTP/2. If you want to articulate this feature in HTTP/1,
+use `maxNumEventLoopsPerHttp1Endpoint` instead.). It indicates that if you change this value `N`,
+then Client instances that are generated by the factory will use distinct `EventLoop` threads per endpoint.
+
+```java
+ClientFactory factory =
+         ClientFactory.builder()
+                      // a client will use maximum 5 EventLoops from the worker group per endpoint
+                      .maxNumEventLoopsPerEndpoint(5)
+                      .build();
+// Do something using factory
+WebClient.builder().factory(factory).build();
+
+ClientFactory factory =
+        ClientFactory.builder()
+                     // or you can just use Integer.MAX_VALUE to use all EventLoops
+                     .maxNumEventLoopsPerEndpoint(Integer.MAX_VALUE)
+                     .build();
+// Do something using factory
+WebClient.builder().factory(factory).build();
+```
+
+The client might need to send many requests to a certain endpoint, while not bothering other endpoints.
+In that situation, you can use <type://ClientFactoryBuilder#maxNumEventLoopsFunction(ToIntFunction)>:
+
+```java
+ClientFactory factory =
+        ClientFactory.builder()
+                     .maxNumEventLoopsFunction(endpoint -> {
+                         if (endpoint.equals(Endpoint.of("lotsOfTraffic.com"))) {
+                             // We are going to use all EventLoops to send reqeusts.
+                             return Integer.MAX_VALUE;
+                         }
+                         // We use just one `EventLoop` per endpoint for the rest.
+                         return 1;
+                     })
+                     .build();
+// Do something using factory
+WebClient.builder().factory(factory).build();
+```
+
+If HTTP/1.1 is used, the client can establish many connections even though only one EventLoop is used.
+It is because of the nature that a client can send only one request at a time using HTTP/1.1 connection,
+if <type://ClientFactoryBuilder#useHttp1Pipelining()> is not enabled.
+You can increase the number of `EventLoop` threads per HTTP/1.1 endpoint to distribute the load
```

Comment:
Wasn't the default for HTTP/1 to distribute connections among event loops?

Reply:
If I understand inside of armeria correctly, one common event loop will handle connections unless user specify the event loop.

Therefore, it would be a choice to distribute the load by increasing the number of threads in event loop.

Reply:
Ah, I forgot to leave a comment. 😅 
Yeah, 1 is the default even for HTTP/1.1.

Reply:
So, it might be reasonable to delete this block :)

Let me delete them.